src/commandHandlers/bio.ts

```diff
@@ -39,6 +39,8 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
             Object.entries((data).bio).forEach(([key, value]) => embed.addField(key.toUpperCase(), value));
         } else {
             embed.addField('Description', 'No bio details found');
+            embed.addField('Example', '^bio description || I am a ...');
+            embed.addField('Example', '^bio location || London, UK');
```

Comment:
Sorry I noticed 1 more thing. `^` can change depending on the deployment.

Can you use `config.COMMAND_PREFIX` instead.

For example...

```typescript
embed.addField('Example', `${config.COMMAND_PREFIX}bio description || I am a ...`);
```